### PR TITLE
Serial tweaks

### PR DESCRIPTION
**Pull request by pelikhan**
- [x] show cursors on status indicator button
- [x] show last value (rounded to 2 sig decimals) in label . The problem is that smoothie only shows min/max not the current value.

### CODE REVIEW

File: webapp/src/serial.tsx [hunk 1]
```diff
@@ -352,7 +353,7 @@ class Chart {
     constructor(source: string, variable: string, value: number, chartIdx: number) {
         const serialTheme = pxt.appTarget.serial && pxt.appTarget.serial.editorTheme
         // Initialize chart
-        const chartConfig = {
+        const chartConfig: IChartOptions = {
```
Comment (caitlin-hennessy) [inline webapp/src/serial.tsx]: Is this interface defined somewhere?

File: webapp/src/serial.tsx [hunk 2]
```diff
@@ -422,7 +422,10 @@ class Chart {
 
     addPoint(value: number) {
         this.line.append(Util.now(), value)
-        this.lastUpdatedTime = Util.now()
+        this.lastUpdatedTime = Util.now();
+        // update label with last value
+        const valueText = Number(Math.round(Number(value + "e+2"))  + "e-2").toString();
```
Comment (caitlin-hennessy) [inline webapp/src/serial.tsx]: What does `Number(Math.round(Number(value + "e+2"))  + "e-2")` do?

File: webapp/src/serial.tsx (hunk 1 above)
Comment (pelikhan) [reply webapp/src/serial.tsx]: yes in our local typings.

File: webapp/src/serial.tsx (hunk 2 above)
Comment (pelikhan) [reply webapp/src/serial.tsx]: It's special JS trick to round number to up to 2 decimal numbers.

**Pull request merged**